perun-core/src/main/java/cz/metacentrum/perun/core/impl/modules/attributes/urn_perun_resource_attribute_def_def_redmineProjectID.java

```diff
@@ -0,0 +1,55 @@
+/*
+ * To change this license header, choose License Headers in Project Properties.
+ * To change this template file, choose Tools | Templates
+ * and open the template in the editor.
+ */
+package cz.metacentrum.perun.core.impl.modules.attributes;
+
+import cz.metacentrum.perun.core.api.Attribute;
+import cz.metacentrum.perun.core.api.AttributeDefinition;
+import cz.metacentrum.perun.core.api.AttributesManager;
+import cz.metacentrum.perun.core.api.Resource;
+import cz.metacentrum.perun.core.api.exceptions.InternalErrorException;
+import cz.metacentrum.perun.core.api.exceptions.WrongAttributeAssignmentException;
+import cz.metacentrum.perun.core.api.exceptions.WrongAttributeValueException;
+import cz.metacentrum.perun.core.api.exceptions.WrongReferenceAttributeValueException;
+import cz.metacentrum.perun.core.impl.PerunSessionImpl;
+import cz.metacentrum.perun.core.implApi.modules.attributes.ResourceAttributesModuleAbstract;
+import cz.metacentrum.perun.core.implApi.modules.attributes.ResourceAttributesModuleImplApi;
+import java.util.regex.Matcher;
+import java.util.regex.Pattern;
+
+/**
+ * Module for redmine project ID
+ * 
+ * @author Peter Balcirak <peter.balcirak@gmail.com>
+ * @date 8.3.2016
+ */
+public class urn_perun_resource_attribute_def_def_redmineProjectID extends ResourceAttributesModuleAbstract implements ResourceAttributesModuleImplApi {
+
+        @Override
+	public void checkAttributeValue(PerunSessionImpl perunSession, Resource resource, Attribute attribute) throws InternalErrorException, WrongAttributeValueException, WrongReferenceAttributeValueException, WrongAttributeAssignmentException {
+		String id = (String) attribute.getValue();
+		if (id == null) {
+			throw new WrongAttributeValueException(attribute, resource, "Attribute can't be empty.");
+		}
+
+		Pattern pattern = Pattern.compile("^[a-z][-_a-z0-9]+$");
+		Matcher match = pattern.matcher(id);
+
+		if (!match.matches()) {
+			throw new WrongAttributeValueException(attribute, resource, "Bad format of attribute redmineProjectID.");

```

Comment:
The exception message will be shown to end users. Please add some information what is the correct format to the message.